locales/en/yourloops.json

```diff
@@ -141,7 +144,7 @@
   "default-min-max": "Default: min at {{min}} and max at {{max}}",
   "default-values": "Default values",
   "delete": "Delete",
-  "delete-medical-report": "Delete medical report",
+  "delete-medical-record": "Delete Medical_record_{{date}}",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Is it not risky to put the beginning of the file name in the translation key?

My point is: maybe the translators will want to translate it (and maybe that's what we want). But if we ever need to display the whole file name and the name is `Medical_record_xxx`, maybe the people in Germany/France/... wouldn't understand? And on the other hand, if we do want to translate it, why putting it as a "raw" file name (with underscores) and not a more user-friendly file name, like `Medical record xxx`? (my questions might be irrelevant as I am really not familiar with this part of the codebase)
‏

Reply:
I do not know the needs but assumed that the file name would need to be translated for all languages :thinking:

Reply:
(I am waiting for Clement's answer)

Reply:
This has been fixed respecting Clement and Aloïs' decision